### PR TITLE
fix: add new section playground; section self-contained

### LINKED ISSUE #271: add self-contained styles for Section
Part of larger #136 for all components.

### PR DESCRIPTION
**Pull request by droshev**
fixes #271 
- create new playground test for `section`
- add self-contained `section.scss`

### CODE REVIEW

File: test/templates/section/index.njk [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+{% from "./component.njk" import section,
+section_headlines %}
```
Comment (greg-a-smith) [inline test/templates/section/index.njk]: Should be on the line above it.

File: scss/layout/_section.scss [hunk 2]
```diff
@@ -13,55 +15,59 @@
...
+    flex: 1;
+    margin-bottom: 0;
+  }
+  &__actions {
+    margin-left: auto;
+  }
+  &__footer {
+    display: flex;
+    justify-content: center;
+  }
```
Comment (greg-a-smith) [inline scss/layout/_section.scss]: It's tough to tell if anything actually changed in this file with the reformatting.  Were there any changes?

File: scss/layout/_section.scss [hunk 3]
```diff
@@ -1,6 +1,8 @@
 @import "./../settings";
 @import "./../functions";
 @import "./../mixins";
+@import "./../core/root";
```
Comment (greg-a-smith) [inline scss/layout/_section.scss]: Why is this being imported?

Comment (droshev) [reply scss/layout/_section.scss]: there are some css variables like `--fd-padding-ui` being defined there

File: scss/layout/_section.scss (hunk 2 above)
Comment (droshev) [reply scss/layout/_section.scss]: I reverted the reformatted code

**Comment by greg-a-smith** [comment]
I created #282 as a follow-up for sections and panels, but it is outside the scope of this PR.

**Pull request merged**